script/grblcon.py

```diff
@@ -129,15 +161,17 @@ class GrblCon:
         'clear': '/0 Clear the screen',
         'close': '/0 Close the current grbl connection',
         'help': '/0 Display this dialog',
-        'home': '/0..4 (X/Y/Z/C) Homes the specified axis. Multiple axis may be provided (or none to home all)',
+        'home': '/0..4 (X/Y/Z/C) Homes the specified axes.',
```

Comment:
The older description had more detail in it. Has the detail been removed because the functionality is gone (for instance, that not specifying an axis no longer homes them all)? If not, why did this change?